Src/PoissonRecon.h

```diff
@@ -510,8 +511,8 @@ template<typename Real>
 void PoissonRecon<Real>::evaluate()
 {
     m_profiler.start();
-    double valueSum = 0;
```

Comment:
Hmmm.  I never really thought about someone using a non-standard type for Real.  I'm wondering how robust some of this code is for that.  There are assumptions that any non-standard Real looks a lot like IEEE floating point.

Thinking about it some more, I think what you've done is fine.  I'll merge as is.

Not sure on the value of the whole <typename Real> thing anyway.  Not sure why anyone would want floats instead of doubles, as even embedded CPUs support doubles today.  I understand it will use more memory, but that seems a small issue compared to run-time.  I believe computation with doubles is marginally faster than with floats on most desktop CPUs.

Thanks again for your help with this.